content/projects/gsoc/2020/project-ideas/jenkins-x-openwrt-ppa.adoc

```diff
@@ -0,0 +1,60 @@
+---
+layout: gsocprojectidea
+title: "PPA for OpenWRT"
+goal: "Create a platform to build and host packages for OpenWRT (PPA, or Personal Package Archive) using Jenkins X"
+category: Jenkins X
+year: 2020
+status: published
```

Comment:
For published ideas we expect to have a quick start guide and links to newbie-friendly issues. I suggest publishing as draft now and updating it later

```suggestion
status: draft
```